### PR TITLE
sidebar scope fix

### PR DESCRIPTION
**Pull request by odedre**
## Proposed Changes

remove sidebar scope component when in scope - drawer now getting TreeNode from outside

### CODE REVIEW

File: components/side-bar/component-tree/component-nodes/component-nodes.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export function ScopeView({ node, depth }: TreeNodeProps<PayloadType>) {
         </div>
       )}
       <AnimateHeight height={collapsed ? 0 : 'auto'}>
-        <div style={indentStyle(depth + 2)} className={classNames(styles.componentTree)}>
```
Comment (KutnerUri) [inline components/side-bar/component-tree/component-nodes/component-nodes.tsx:29]: so you gave up on the double indent?

File: components/side-bar/component-tree/index.ts [hunk 2]
```diff
@@ -1 +1,6 @@
 export * from './component-tree';
+export * from './recursive-tree';
+export { ComponentView } from './component-view';
+export { ScopeView, NamespaceView } from './component-nodes';
```
Comment (KutnerUri) [inline components/side-bar/component-tree/index.ts:4]: plz split. What do you think about ScopeTreeNode and NamespaceTreeNode?

File: components/side-bar/index.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
 export { ComponentTree } from './component-tree';
 export { Collapser } from './sidebar-collapser';
 export { StatusTypes } from './component-tree/recursive-tree';
+export { ComponentView, ScopeView, NamespaceView, PayloadType, TreeNodeProps, ScopePayload } from './component-tree';
```
Comment (KutnerUri) [inline components/side-bar/index.ts:4]: merge exports

File: extensions/component-tree/component-tree.drawer.tsx [hunk 4]
```diff
@@ -1,20 +1,20 @@
-import { useComponentHost } from '@teambit/component';
-import { Drawer } from '@teambit/sidebar';
-import { FullLoader } from '@teambit/staged-components.full-loader';
-import { ComponentTree } from '@teambit/staged-components.side-bar';
-import React from 'react';
+// import { useComponentHost } from '@teambit/component';
```
Comment (KutnerUri) [inline extensions/component-tree/component-tree.drawer.tsx:1]: is this supposed to be commented out?

File: extensions/scope/components.drawer.tsx [hunk 5]
```diff
@@ -12,7 +19,31 @@ export class ComponentsDrawer implements Drawer {
...
     const { scope } = useScope();
+    const { treeNodeSlot } = this;
+
+    const TreeNodeRenderer = useCallback(
+      function TreeNode(props: TreeNodeProps<PayloadType>) {
+        const children = props.node.children;
+
+        if (!children) return <ComponentView {...props} treeNodeSlot={treeNodeSlot} />;
+
+        // skip over scope node and render only children
```
Comment (KutnerUri) [inline extensions/scope/components.drawer.tsx:30]: 👍

**Review by KutnerUri — CHANGES_REQUESTED** [review]

**Comment by KutnerUri** [comment]
looks good!

**Review by KutnerUri — APPROVED** [review]
looks good

**Review by guysaar223 — APPROVED** [review]

**Pull request merged**